pkg/apis/core/types.go

```diff
@@ -51,6 +51,9 @@ type Volume struct {
 	// This implied behavior is deprecated and will be removed in a future version.
 	// +optional
 	VolumeSource
+	// For unit testing
+	// GCEPersistentDisk Whether the test meets the expectation
+	GCEPersistentDisk *GCEPersistentDiskVolumeSource
```

Comment:
Sorry, can you explain this?  We can't do this here.

Reply:
I used it for unit testing, and I will delete it immediately

Reply:
I have found GCEPersistentDisk for unit testing, which I didn't notice before

Reply:
I need to get `GCEPersistentDisk` for unit testing,I  in `volumesource` I found it in